From 4e543f422750d200536325b699769550b4bfa19e Mon Sep 17 00:00:00 2001
From: Manikanta Mylavarapu <quic_mmanikan@quicinc.com>
Date: Wed, 19 Jun 2024 21:25:09 +0530
Subject: [PATCH] linux: drivers: ipq5424: replace devsoc with ipq5424

Replce 'devsoc' with 'ipq5424' in driver compatibles, data structures,
kconfig, makefiles etc.

Change-Id: Iccf16ea258d890971bd27b3552aa8d9e1cd7e5bc
Signed-off-by: Manikanta Mylavarapu <quic_mmanikan@quicinc.com>
---
 arch/arm/mach-qcom/Kconfig             |  4 ++--
 drivers/clk/qcom/Kconfig               |  8 ++++----
 drivers/clk/qcom/Makefile              |  2 +-
 drivers/mtd/nand/raw/qcom_nandc.c      |  6 +++---
 drivers/pci/controller/dwc/pcie-qcom.c |  2 +-
 drivers/pinctrl/qcom/Kconfig.msm       |  8 ++++----
 drivers/pinctrl/qcom/Makefile          |  2 +-
 drivers/soc/qcom/llcc-qcom.c           | 10 +++++-----
 8 files changed, 21 insertions(+), 21 deletions(-)

diff --git a/arch/arm/mach-qcom/Kconfig b/arch/arm/mach-qcom/Kconfig
index eff43516694e..d2d0542b45c4 100644
--- a/arch/arm/mach-qcom/Kconfig
+++ b/arch/arm/mach-qcom/Kconfig
@@ -17,8 +17,8 @@ config ARCH_IPQ40XX
 	select CLKSRC_QCOM
 	select HAVE_ARM_ARCH_TIMER
 
-config ARCH_DEVSOC
-	bool "Enable support for DEVSOC"
+config ARCH_IPQ5424
+	bool "Enable support for IPQ5424"
 	select ARM_GIC_V3
 
 config ARCH_MSM8X60
diff --git a/drivers/clk/qcom/Kconfig b/drivers/clk/qcom/Kconfig
index 3bdbbc51fe3e..a6a73bea3aac 100644
--- a/drivers/clk/qcom/Kconfig
+++ b/drivers/clk/qcom/Kconfig
@@ -161,10 +161,10 @@ config IPQ_GCC_5332
 	  Say Y if you want to use peripheral devices such as UART, SPI,
 	  i2c, USB, SD/eMMC, etc.
 
-config IPQ_GCC_DEVSOC
-	tristate "DEVSOC Global Clock Controller"
+config IPQ_GCC_5424
+	tristate "IPQ5424 Global Clock Controller"
 	help
-	  Support for the global clock controller on devsoc devices.
+	  Support for the global clock controller on ipq5424 devices.
 	  Say Y if you want to use peripheral devices such as UART, SPI,
 	  i2c, USB, SD/eMMC, etc.
 
@@ -224,7 +224,7 @@ config IPQ_NSSCC_9574
 
 config IPQ_NSSCC_DEVSOC
         tristate "DEVSOC NSS Clock Controller"
-        depends on IPQ_GCC_DEVSOC
+        depends on IPQ_GCC_5424
         help
           Support for NSS clock controller on devsoc devices.
 
diff --git a/drivers/clk/qcom/Makefile b/drivers/clk/qcom/Makefile
index 7eb4d9b83e15..a2715447778d 100644
--- a/drivers/clk/qcom/Makefile
+++ b/drivers/clk/qcom/Makefile
@@ -26,7 +26,7 @@ obj-$(CONFIG_IPQ_APSS_6018) += apss-ipq6018.o
 obj-$(CONFIG_IPQ_GCC_4019) += gcc-ipq4019.o
 obj-$(CONFIG_IPQ_GCC_5018) += gcc-ipq5018.o
 obj-$(CONFIG_IPQ_GCC_5332) += gcc-ipq5332.o
-obj-$(CONFIG_IPQ_GCC_DEVSOC) += gcc-devsoc.o
+obj-$(CONFIG_IPQ_GCC_5424) += gcc-ipq5424.o
 obj-$(CONFIG_IPQ_GCC_6018) += gcc-ipq6018.o
 obj-$(CONFIG_IPQ_GCC_806X) += gcc-ipq806x.o
 obj-$(CONFIG_IPQ_GCC_8074) += gcc-ipq8074.o
diff --git a/drivers/mtd/nand/raw/qcom_nandc.c b/drivers/mtd/nand/raw/qcom_nandc.c
index 2d474a5a742d..b380d074a765 100644
--- a/drivers/mtd/nand/raw/qcom_nandc.c
+++ b/drivers/mtd/nand/raw/qcom_nandc.c
@@ -4360,7 +4360,7 @@ static const struct qcom_nandc_props ipq5332_nandc_props = {
 	.dev_cmd_reg_start = 0x7000,
 };
 
-static const struct qcom_nandc_props devsoc_nandc_props = {
+static const struct qcom_nandc_props ipq5424_nandc_props = {
 	.ecc_modes = (ECC_BCH_4BIT | ECC_BCH_8BIT),
 	.is_bam = true,
 	.is_serial_nand = true,
@@ -4408,8 +4408,8 @@ static const struct of_device_id qcom_nandc_of_match[] = {
 		.data = &ipq5332_nandc_props,
 	},
 	{
-		.compatible = "qcom,devsoc-nand",
-		.data = &devsoc_nandc_props,
+		.compatible = "qcom,ipq5424-nand",
+		.data = &ipq5424_nandc_props,
 	},
 	{
 		.compatible = "qcom,ipq9574-nand",
diff --git a/drivers/pci/controller/dwc/pcie-qcom.c b/drivers/pci/controller/dwc/pcie-qcom.c
index 7b6fd71bf611..e75754e6d7dd 100644
--- a/drivers/pci/controller/dwc/pcie-qcom.c
+++ b/drivers/pci/controller/dwc/pcie-qcom.c
@@ -1920,7 +1920,7 @@ static const struct of_device_id qcom_pcie_match[] = {
 	{ .compatible = "qcom,pcie-apq8084", .data = &cfg_1_0_0 },
 	{ .compatible = "qcom,pcie-ipq4019", .data = &cfg_2_4_0 },
 	{ .compatible = "qti,pcie-ipq5332", .data = &cfg_1_27_0 },
-	{ .compatible = "qcom,pcie-devsoc", .data = &cfg_1_27_0 },
+	{ .compatible = "qcom,pcie-ipq5424", .data = &cfg_1_27_0 },
 	{ .compatible = "qcom,pcie-ipq6018", .data = &cfg_2_9_0 },
 	{ .compatible = "qcom,pcie-ipq8064", .data = &cfg_2_1_0 },
 	{ .compatible = "qcom,pcie-ipq8064-v2", .data = &cfg_2_1_0 },
diff --git a/drivers/pinctrl/qcom/Kconfig.msm b/drivers/pinctrl/qcom/Kconfig.msm
index 71b0ffa76a64..d2c01085f15e 100644
--- a/drivers/pinctrl/qcom/Kconfig.msm
+++ b/drivers/pinctrl/qcom/Kconfig.msm
@@ -45,13 +45,13 @@ config PINCTRL_IPQ5332
 	  Qualcomm Technologies Inc TLMM block found on the Qualcomm
 	  Technologies Inc IPQ5332 platform.
 
-config PINCTRL_DEVSOC
-	tristate "Qualcomm Technologies, Inc. DEVSOC pin controller driver"
+config PINCTRL_IPQ5424
+	tristate "Qualcomm Technologies, Inc. IPQ5424 pin controller driver"
 	help
 	  This is the pinctrl, pinmux, pinconf and gpiolib driver for
           the Qualcomm Technologies Inc. TLMM block found on the
-          Qualcomm Technologies Inc. DEVSOC platform. Select this for
-          DEVSOC.
+          Qualcomm Technologies Inc. IPQ5424 platform. Select this for
+          IPQ5424.
 
 config PINCTRL_IPQ8074
 	tristate "Qualcomm Technologies, Inc. IPQ8074 pin controller driver"
diff --git a/drivers/pinctrl/qcom/Makefile b/drivers/pinctrl/qcom/Makefile
index ec241e80781b..5c686d4c8ef3 100644
--- a/drivers/pinctrl/qcom/Makefile
+++ b/drivers/pinctrl/qcom/Makefile
@@ -7,7 +7,7 @@ obj-$(CONFIG_PINCTRL_IPQ4019)	+= pinctrl-ipq4019.o
 obj-$(CONFIG_PINCTRL_IPQ5018)	+= pinctrl-ipq5018.o
 obj-$(CONFIG_PINCTRL_IPQ8064)	+= pinctrl-ipq8064.o
 obj-$(CONFIG_PINCTRL_IPQ5332)	+= pinctrl-ipq5332.o
-obj-$(CONFIG_PINCTRL_DEVSOC)	+= pinctrl-devsoc.o
+obj-$(CONFIG_PINCTRL_IPQ5424)	+= pinctrl-ipq5424.o
 obj-$(CONFIG_PINCTRL_IPQ8074)	+= pinctrl-ipq8074.o
 obj-$(CONFIG_PINCTRL_IPQ6018)	+= pinctrl-ipq6018.o
 obj-$(CONFIG_PINCTRL_IPQ9574)	+= pinctrl-ipq9574.o
diff --git a/drivers/soc/qcom/llcc-qcom.c b/drivers/soc/qcom/llcc-qcom.c
index 6cf8c1c9f888..2e1e1fb6d556 100644
--- a/drivers/soc/qcom/llcc-qcom.c
+++ b/drivers/soc/qcom/llcc-qcom.c
@@ -357,7 +357,7 @@ static const struct llcc_slice_config sm8550_data[] =  {
 	{LLCC_VIDVSP,   28,  256, 4, 1, 0xFFFFFF, 0x0,   0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, },
 };
 
-static const struct llcc_slice_config devsoc_data[] =  {
+static const struct llcc_slice_config ipq5424_data[] =  {
 	/* SCID 1 - Descriptors (PPE, CUMAC) */
 	{LLCC_CPUSS,     1,  768, 1, 0,  0xFFFF, 0x0,  0, 0, 0, 1, 1, 0, 1, 1, 1, 0, 0, 1, 1, 1, 1, },
 	/* SCID 2 - SKB, SKB data (PPE, CUMAC, APSS Core 0,1,2) */
@@ -527,9 +527,9 @@ static const struct qcom_llcc_config sm8550_cfg = {
 	.edac_reg_offset = &llcc_v2_1_edac_reg_offset,
 };
 
-static const struct qcom_llcc_config devsoc_cfg = {
-	.sct_data       = devsoc_data,
-	.size           = ARRAY_SIZE(devsoc_data),
+static const struct qcom_llcc_config ipq5424_cfg = {
+	.sct_data       = ipq5424_data,
+	.size           = ARRAY_SIZE(ipq5424_data),
 	.need_llcc_cfg	= true,
 	.reg_offset	= llcc_v2_1_reg_offset,
 	.edac_reg_offset = &llcc_v2_1_edac_reg_offset,
@@ -1079,7 +1079,7 @@ static int qcom_llcc_probe(struct platform_device *pdev)
 }
 
 static const struct of_device_id qcom_llcc_of_match[] = {
-	{ .compatible = "qcom,devsoc-llcc", .data = &devsoc_cfg },
+	{ .compatible = "qcom,ipq5424-llcc", .data = &ipq5424_cfg },
 	{ .compatible = "qcom,sc7180-llcc", .data = &sc7180_cfg },
 	{ .compatible = "qcom,sc7280-llcc", .data = &sc7280_cfg },
 	{ .compatible = "qcom,sc8180x-llcc", .data = &sc8180x_cfg },
-- 
2.34.1

